tests/baselines/reference/user/TypeScript-Node-Starter.log

```diff
@@ -0,0 +1,7 @@
+Exit Code: 1
+Standard output:
+node_modules/@types/passport-facebook/index.d.ts(50,31): error TS2689: Cannot extend an interface 'passport.Strategy'. Did you mean 'implements'?
```

Comment:
@andy-ms, can you take a look at what happened here?

Reply:
I don't see this with a fresh `npm install`. Do you know what command the bot runs?

Reply:
I did:

* `jake clean`
* `jake local`
* `git submodule update tests/cases/user/TypeScript-Node-Starter/`
* `cd tests/cases/user/TypeScript-Node-Starter/TypeScript-Node-Starter`
* `rm -r node_modules`
* `npm i --no-save --ignore-scripts`
* `node ../../../../../built/local/tsc.js`

I see compile errors in `mocha`, `jquery`, and `lodash`. `@types/passport-facebook` is installed but I don't see any compile errors there.